test/cli_create_test.go

```diff
@@ -33,37 +33,6 @@ func (suite *PouchCreateSuite) SetUpSuite(c *check.C) {
 func (suite *PouchCreateSuite) TearDownTest(c *check.C) {
 }
 
-// TestCreateName is to verify the correctness of creating container with specified name.
-func (suite *PouchCreateSuite) TestCreateName(c *check.C) {
-	name := "create-normal"
-	res := command.PouchRun("create", "--name", name, busyboxImage)
-
-	res.Assert(c, icmd.Success)
-	if out := res.Combined(); !strings.Contains(out, name) {
```

Comment:
No need to remove all the test case. Maybe we just remove the string comparing.

Reply:
That sounds reasonable. Thx.